src/BaseCurve.sol

```diff
@@ -55,19 +68,19 @@ abstract contract BaseCurve {
     /// @param totalShares Total quantity of shares already awarded by the curve
     /// @return shares The number of shares that would need to be redeemed
     function previewWithdraw(uint256 assets, uint256 totalAssets, uint256 totalShares)
-        public
+        external
         view
         virtual
         returns (uint256 shares);
 
-    /// @notice Preview how many assets would be required to mint a specific amount of shares
+    /// @notice Preview how many assets would be returned for burning a specific amount of shares
     ///
-    /// @param shares Quantity of shares to mint
+    /// @param shares Quantity of shares to burn
     /// @param totalShares Total quantity of shares already awarded by the curve
     /// @param totalAssets Total quantity of assets already staked into the curve
-    /// @return assets The number of assets that would be required to mint the shares
-    function previewMint(uint256 shares, uint256 totalShares, uint256 totalAssets)
-        public
```

Comment:
3.20 - Functions in Curve Contracts Could Be external Instead of public

Made this `external`